tests/SpecTests/CommandExpectations.php

```diff
@@ -2,7 +2,6 @@
 
 namespace MongoDB\Tests\SpecTests;
 
-use MongoDB\BSON\Timestamp;
```

Comment:
This was leftover from some earlier wip commit in a previous PR when the command document assertion was done within this class. That logic was ultimately moved to `TransactionsSpecTest::assertCommandMatches()`.